.vscode/settings.json

```diff
@@ -0,0 +1,7 @@
+{
+    "spellright.language": [],
```

Comment:
Is `spellright` an extension? If so, I would prefer to not make changes to this repo that are specific to a particular extension...not without further discussion/consideration/support at least.

Reply:
Oops this was a mistake! Completely agree.